client/lib/plans/personal-plan.js

```diff
@@ -32,18 +44,33 @@ export const personalPlan = {
 	available: 'yes',
 	bundle_product_ids: [ 12, 9, 50, 5, 6, 46, 54, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 72, 73, 74, 75, 16 ],
 	bill_period_label: translate( 'per year' ),
-	price: '$71.88',
-	formatted_price: '$71.88',
-	raw_price: 71.88,
 	has_domain_credit: true
 };
 
+const getCurrencyCode = plans => head( map( plans, property( 'currency_code' ) ) ) || 'USD';
+
+const applyCurrency = currencyCode => {
+	const cost = personalPlan.prices[ currencyCode ];
+	const price = formatCurrency( cost, currencyCode );
+
+	return {
+		...personalPlan,
+		cost,
+		price,
+		raw_price: cost,
+		formatted_price: price,
+		currency_code: currencyCode
+	};
+};
+
+const formatPlan = flow( getCurrencyCode, applyCurrency );
```

Comment:
The _flow_ here is a bit confusing to me. It appears like we start with a list of plans, grab the currency for the first plan, then return the object for the personal plan. Is this right?

If that's the case, why bother with the `map` and `head`? Why not just grab the currency code for the personal plan etc… I would expect code written this way to take a list of plans, then return _a list of plans_ with the currency information applied.